applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_line_interface_element.cpp

```diff
@@ -0,0 +1,560 @@
+
+    Model model;
+    auto element = CreateHorizontalUnitLength3Plus3NodedLineInterfaceElementWithDisplacementDoF(model, p_properties);
+
+    const auto dummy_process_info = ProcessInfo{};
+    element.Initialize(dummy_process_info);
+
+    element.GetGeometry()[3].FastGetSolutionStepValue(DISPLACEMENT) = array_1d<double, 3>{0.2, 0.5, 0.0};
+    element.GetGeometry()[4].FastGetSolutionStepValue(DISPLACEMENT) = array_1d<double, 3>{0.2, 0.5, 0.0};
+    element.GetGeometry()[5].FastGetSolutionStepValue(DISPLACEMENT) = array_1d<double, 3>{0.2, 0.5, 0.0};
+
+    // Act
+    Vector actual_right_hand_side;
+    Matrix actual_left_hand_side;
+    element.CalculateLocalSystem(actual_left_hand_side, actual_right_hand_side, dummy_process_info);
+
+    // Assert
+    auto expected_left_hand_side    = Matrix{ZeroMatrix{12, 12}};
+    expected_left_hand_side(0, 0)   = shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(1, 1)   = normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(2, 2)   = shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(3, 3)   = normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(4, 4)   = shear_stiffness * (2.0 / 3.0);
+    expected_left_hand_side(5, 5)   = normal_stiffness * (2.0 / 3.0);
+    expected_left_hand_side(6, 6)   = shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(7, 7)   = normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(8, 8)   = shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(9, 9)   = normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(10, 10) = shear_stiffness * (2.0 / 3.0);
+    expected_left_hand_side(11, 11) = normal_stiffness * (2.0 / 3.0);
+
+    expected_left_hand_side(0, 6)  = -shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(1, 7)  = -normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(2, 8)  = -shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(3, 9)  = -normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(4, 10) = -shear_stiffness * (2.0 / 3.0);
+    expected_left_hand_side(5, 11) = -normal_stiffness * (2.0 / 3.0);
+
+    expected_left_hand_side(6, 0)  = -shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(7, 1)  = -normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(8, 2)  = -shear_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(9, 3)  = -normal_stiffness * (1.0 / 6.0);
+    expected_left_hand_side(10, 4) = -shear_stiffness * (2.0 / 3.0);
+    expected_left_hand_side(11, 5) = -normal_stiffness * (2.0 / 3.0);
+
+    KRATOS_EXPECT_MATRIX_RELATIVE_NEAR(actual_left_hand_side, expected_left_hand_side, Defaults::relative_tolerance)
+
+    auto expected_right_hand_side = Vector{ZeroVector{12}};
+    expected_right_hand_side <<= 0.33333333, 1.66666667, 0.33333333, 1.66666667, 1.33333333,
+        6.66666667, -0.33333333, -1.66666667, -0.33333333, -1.66666667, -1.33333333, -6.66666667;
```

Comment:
Should we use this or write 1/3, 5/3, 1/3, 5/3, 4/3, 19/3, -1/3 et.c

Similarly maybe at line 429, 463 and the similar things where square roots of 3 are involved

Guess this is a matter of preference.

Reply:
For this set of 'n/3' I completely agree, if I remember correctly the 429 and 463 examples, the analytical solution was not a lot more clear than just the numbers generated by Annes python 'check' script. 

I'll wait for the sonarcloud results here: if I need to make changes because of that, I'll change this line, otherwise I'd propose to merge this and avoid another pipeline wait.